CODEOWNERS

```diff
@@ -2,37 +2,28 @@
 # 1. Per Github docs: "Order is important; the last matching pattern takes the most precedence."
```

Comment:
The errors in this file result because mercury-team and devrel don't have write access. That should be updated.